presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1805,6 +1807,17 @@ public SystemSessionProperties(
                         "The regexp of traced task id. We only enable trace on a task if its id matches.",
                         "",
                         false),
+                longProperty(NATIVE_MAX_OUTPUT_BUFFER_SIZE,
+                        "The maximum size in bytes for the task's buffered output. The buffer is shared among all drivers.",
+                        200L << 20,
+                        false),
+                longProperty(NATIVE_MAX_PAGE_PARTITIONING_BUFFER_SIZE,
+                        "The maximum bytes to buffer per PartitionedOutput operator to avoid creating tiny " +
+                                "SerializedPages. For PartitionedOutputNode::Kind::kPartitioned, PartitionedOutput operator " +
+                                "would buffer up to that number of bytes / number of destinations for each destination before " +
+                                "producing a SerializedPage.",
```

Comment:
align on left

Reply:
The checkstyle does not seem to like it when I tried to align the concatenated strings left..